lib/server.js

```diff
@@ -15,7 +15,9 @@
 
 'use strict';
 
-var _ = require('lodash');
+var isString = require('lodash/isString');
```

Comment:
@remarkablemark Yeah, as an optimization it's absolutely premature. As a style and consistency issue, it's kinda minor. I wouldn't hold up a merge for something like this.